src/status_im2/contexts/onboarding/identifiers/view.cljs

```diff
@@ -0,0 +1,62 @@
+(ns status-im2.contexts.onboarding.identifiers.view
+  (:require [react-native.core :as rn]
+            [clojure.string :as string]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]
+
+            [status-im2.contexts.onboarding.identifiers.style :as style]
+            [status-im2.contexts.onboarding.common.background.view :as background]
+            [status-im2.contexts.onboarding.common.carousel.view :as carousel]
+            [status-im2.contexts.onboarding.common.carousel.animation :as carousel.animation]))
+
+(def header-text
+  [{:text     (i18n/label :t/unique-identifiers)
+    :sub-text (i18n/label :t/your-identifiers)}
+   {:text     (i18n/label :t/identicon-ring)
+    :sub-text (i18n/label :t/identicon-ring-explanation)}
+   {:text     (i18n/label :t/chat-key-title)
+    :sub-text (i18n/label :t/chat-key-description)}
+   {:text     (i18n/label :t/emojihash)
+    :sub-text (i18n/label :t/emojihash-description)}])
+
+(defn view
+  []
+  (let [progress                                                         (reagent/atom nil)
+        paused                                                           (reagent/atom nil)
+        {:keys [emoji-hash display-name profile-picture compressed-key]} (rf/sub [:multiaccount])
+        {:keys [color]}                                                  (rf/sub [:onboarding-2/profile])
+        emoji-string                                                     (string/join emoji-hash)]
+    [:f>
+     (fn []
+       (carousel.animation/initialize-animation progress paused)
+       (fn []
+         [:<>
+          [background/view true]
```

Comment:
This looks weird, background also has carousel and this screen also adds it.

Reply:
yes but this is how the page is expected to be, i.e the onboarding background uses the carousel and this page also uses a carousel and the onboarding background  :)